package.json

```diff
@@ -16,15 +16,53 @@
     "apps/*"
   ],
   "devDependencies": {
+    "@babel/core": "^7.14.5",
+    "@babel/preset-react": "^7.14.5",
     "@commitlint/cli": "^18.0.0",
     "@commitlint/config-conventional": "^18.0.0",
+    "@leather-wallet/eslint-config": "workspace:^",
```

Comment:
I was testing out `Nx` react library generators to create the `tokens` package but it adds a lot of extras here we don't really want / need. 

I'll rework this and add the package manually which is actually easier than using the generator and then cleaning up afterwards.

We can consider creating our own generator in the future if we need to but I think am updated `README` with Copy+paste style tips would suffice